canvas/fizz-21-py/src/test.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python_defaultSpec_1598852742975",
+   "display_name": "Python 3.8.5 32-bit ('py-spike': pipenv)"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FIZZ-21 Generate visualizations with the data from canvas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Basic configuration and import csv files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from dotenv import load_dotenv\n",
+    "import os\n",
+    "\n",
+    "load_dotenv()\n",
+    "os.chdir(os.getenv(\"CSV_INPUT_BASE_PATH\"))\n",
```

Comment:
As a general rule, I would put the full path into the `read_csv` command instead of changing the local working directory. When changing the working directory, it is best to restore to the original location. If the program crashes, then it might not get to the point of changing back to that original location. Thus best not to change locations.